<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>450</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Accerman function</string>
  </property>
  <property name="windowIcon">
   <iconset theme="accessories-calculator"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="computeAccermanFuncPushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>294</y>
      <width>351</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Compute Accerman Function</string>
    </property>
   </widget>
   <widget class="QLabel" name="wordResultLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>190</y>
      <width>71</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Result:</string>
    </property>
   </widget>
   <widget class="QLabel" name="mArgumentLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>M argument:</string>
    </property>
   </widget>
   <widget class="QLabel" name="nArgumentLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>N argument:</string>
    </property>
   </widget>
   <widget class="QLabel" name="resultLabel">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>180</y>
      <width>301</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="mArgumentLineEdit">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>60</y>
      <width>250</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="nArgumentLineEdit">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>120</y>
      <width>250</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="attentionLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>781</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>ATTENTION! Accerman function grows fast even on low numbers so range of arguments is m (0 to 3) and n (0 to 10)</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
